src/header/Header.tsx

```diff
@@ -69,8 +73,9 @@ const Header = ({
         mainMenuDropdowns={mainMenuDropdowns}
         outlineLink={outlineLink}
         searchButtonAction={meiliSearchEnabled ? openSearchModal : undefined}
+        containerProps={containerProps}
```

Comment:
Do we actually want different pages/sections of this Authoring app to show a differently-sized header? This is probably a question for the CCs with oversight over the whole app, but I would think it's better to have [these params](https://github.com/openedx/frontend-app-authoring/pull/1258/files#diff-b607a6130d15e58997aa9499ad9dfa67e094b17d966b5750e86f5bb7cadaaa73R165-R167) here, and remove the `containerProps` from this Header class, so other pages don't mess with them.

```suggestion
        containerProps={{
            size: undefined,
        }}
```

Reply:
> Do we actually want different pages/sections of this Authoring app to show a differently-sized header?

The idea came from here: https://github.com/openedx/frontend-app-authoring/issues/1221#issuecomment-2322103512
And there is a previous discussion here: https://github.com/openedx/frontend-app-authoring/pull/1217#issuecomment-2310702864
Personally, I prefer consistency throughout the app. But we would definitely use the extra space in the library authoring, and maybe changing everything would be a too big step.